tools/clang/tools/dotnetc/EditorForm.Designer.cs

```diff
@@ -631,47 +631,61 @@ private void InitializeComponent()
             "ps_6_4",
             "ps_6_5",
             "ps_6_6",
+            "ps_6_7",
+            "ps_6_8",
             "vs_6_0",
             "vs_6_1",
             "vs_6_2",
             "vs_6_3",
             "vs_6_4",
             "vs_6_5",
             "vs_6_6",
+            "vs_6_7",
+            "vs_6_8",
             "cs_6_0",
             "cs_6_1",
             "cs_6_2",
             "cs_6_3",
             "cs_6_4",
             "cs_6_5",
             "cs_6_6",
+            "cs_6_7",
+            "cs_6_8",
             "gs_6_0",
             "gs_6_1",
             "gs_6_2",
             "gs_6_3",
             "gs_6_4",
             "gs_6_5",
             "gs_6_6",
+            "gs_6_7",
+            "gs_6_8",
             "hs_6_0",
             "hs_6_1",
             "hs_6_2",
             "hs_6_3",
             "hs_6_4",
             "hs_6_5",
             "hs_6_6",
+            "hs_6_7",
+            "hs_6_8",
             "ds_6_0",
             "ds_6_1",
             "ds_6_2",
             "ds_6_3",
             "ds_6_4",
             "ds_6_5",
             "ds_6_6",
+            "ds_6_7",
+            "ds_6_8",
```

Comment:
Perhaps we should add `ms` and `as`, and remove `lib_6_1` and `lib_6_2`, since these are not valid targets.

Reply:
Can we add `lib_6_x` as well?